lib/SPIRV/libSPIRV/SPIRVType.h

```diff
@@ -403,6 +403,8 @@ class SPIRVTypeImage:public SPIRVType {
   SPIRVCapVec getRequiredCapability() const {
     SPIRVCapVec CV;
     CV.push_back(CapabilityImageBasic);
+    if (Desc.Dim == 1)
+      CV.push_back(CapabilitySampled1D);

```

Comment:
It's required by specification and checked by SPIR-V validation tool.
Dependency on Shader capability looks odd to me.
I'll revert this change while it's not cleared up.
Thanks!